### PR TITLE
Support for adding episodes to post

### PR DESCRIPTION
**Pull request by duvholt**

### CODE REVIEW

File: data_models/models.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ class Episode(models.Model):
     class Meta:
         verbose_name = "Episode"
         verbose_name_plural = "Episoder"
+        ordering = ['-created_at']
```
Comment (santi) [inline data_models/models.py]: This change is not reflected in the last migration

File: data_models/models.py [hunk 2]
```diff
@@ -119,6 +120,7 @@ class Post(models.Model):
     class Meta:
         verbose_name = "Artikkel"
         verbose_name_plural = "Artikler"
+        ordering = ['-publish_at']
```
Comment (santi) [inline data_models/models.py]: This change is not reflected in the last migration

File: data_models/models.py [hunk 3]
```diff
@@ -127,6 +129,13 @@ class Meta:
     content = models.TextField('Brødtekst')
     deleted = models.BooleanField('Slettet', default=False)
 
+    episodes = models.ManyToManyField(
+        Episode,
+        blank=True,
+        verbose_name='Episoder',
+        help_text='Episoder blir sortert på program hvis artikkelen er lagret.'
```
Comment (santi) [inline data_models/models.py]: I don't understand the meaning of the help_text, perhaps "Legg til episoder som vises nederst i artikkelen"

**Comment by duvholt** [comment]
Good idea

**Comment by duvholt** [comment]
I squashed the help text fix, but the other fix commit doesn't really fit into any other commit since it depends both on ordering and help text.

**Pull request merged**